extensions/functions_arithmetic.yaml

```diff
@@ -173,6 +207,68 @@ scalar_functions:
           - value: i64
           - value: i64
         return: i64
+  -
+    name: "power"
+    description: "Take the power with the first value as the base and second as exponent"
+    impls:
+      - args:
```

Comment:
Where did you come up with these signatures from? For example, why does i8^i8 return i8? It seems like we'll run out of space very quickly. I feel like many systems support something like `power(i64, fp64) => fp64` and `power(fp64,fp64) => fp64` and that is enough.

Reply:
@jacques-n do you have a strong preference which direction we go here?  Should we have multiple `power` functions?